ms_gsl.sh

```diff
@@ -0,0 +1,31 @@
+package: ms_gsl
+version: master
+source: https://github.com/Microsoft/GSL.git
+tag: master
+---
+#!/bin/bash -e
+
+# recipe for the C++ guidelines support library (Microsoft implementation)
+# can be deleted once we are fully C++17 compliant
+
+# just rsync into the installdir
+rsync -a $SOURCEDIR/ $INSTALLROOT/
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
```

Comment:
Why do you need this? `gsl` is header only, so there is no need to for the modulefile.

Reply:
ok